components/Tags/Tags.styles.ts

```diff
@@ -8,42 +8,27 @@ import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
 export const tagsStyles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
-      marginTop: '1rem',
-      marginBottom: '1rem',
+      display: 'flex',
+      alignItems: 'center',
+      flexWrap: 'wrap',
+      margin: '.5rem 0',
```

Comment:
Spacing values should be calculated with theme helpers, ``margin: `${theme.typography.pxToRem(theme.spacing(1))} 0`, ``

Reply:
Fixed!